src/Components/HomePage/Homepage.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './Homepage'
+
+export function Homepage() {
+    return (
+        <main>
+            <article>HOLA</article>
+            <section>
+                <article>Adult Activities</article>
```

Comment:
We're probably going to want to have `heading` tags within each of these articles to display the text.